tests/foreman/api/test_errata.py

```diff
@@ -55,6 +54,8 @@
 
 CUSTOM_REPO_URL = FAKE_6_YUM_REPO
 CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+DISTRO_RHEL6 = settings.distro.image_el6
```

Comment:
Ok, another idea - if settings are already initialized on import level, please take a look whether they're already initialized while importing from `constants.py`. If they're initialized - instead of dropping `DISTRO_RHEL6` and `DISTRO_RHEL7` from constants.py you could set the value from settings file inside `contants.py`. This way you won't need any changes in test modules, distros will be loaded like before.